### PR TITLE
Make it clear in the readme that the fake account api is provided

### PR DESCRIPTION
**Pull request by hallabro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,9 @@ before reaching out for help
 - Use (copy or otherwise) code from any third party library without attribution to complete the exercise, as this will result in the test being rejected
```
Comment (stevecookform3) [inline README.md:23]: > 'third party library'
i'd just change this to 'third party', since we discourage plagiarism from any source

File: README.md [hunk 2]
```diff
@@ -1,29 +1,31 @@
...
 
 #### Docker-compose
  - Add your solution to the provided docker-compose file
  - We should be able to run `docker-compose up` and see your tests run against the provided account API service 
 
 ### Please don't
 - Use a code generator to write the client library
-- Use (copy or otherwise) code from any third party library without attribution to complete the exercise, as this will result in the test being rejected
+- Use (copy or otherwise) code from any third party without attribution to complete the exercise, as this will result in the test being rejected
 - Use a library for your client (e.g: go-resty). Only test libraries are allowed.
```
Comment (diogovieira) [inline README.md:24]: Hijacking this a bit because it's not a change in the pull request but I guess this is as good as any other opportunity to bring this up.

The wording here seems to imply candidates can't use any third-party library whatsoever, which I _think_ it's not right. We just don't want them to use one that would implement the client with generated code or something similar, right?

I remember one specific instance where this _may_ have lead a candidate to implement his own UUID generation code which I think it's not ideal since we wouldn't do that in production code anyway for multiple reasons. Would it make sense to reword this to make it clearer? Or am I making a wrong assumption here?

Again, sorry for hijacking this @hallabro 😄

**Review by stevecookform3 — APPROVED** [review]
